lxd/storage_pools.go

```diff
@@ -624,7 +624,7 @@ func storagePoolGet(d *Daemon, r *http.Request) response.Response {
 		poolAPI.Status = pool.LocalStatus()
 	}
 
-	etag := []any{pool.Name, pool.Driver, poolAPI.Config}
+	etag := []any{pool.Name(), pool.Driver().Info().Name, poolAPI.Config}
```

Comment:
I think we should use fields from poolAPI: Name, Config, Description

Reply:
I made it equal as here in `storagePoolPut()`:

https://github.com/canonical/lxd/blob/7957bdf5541a4169c676f5ebb300b9ad857957c5/lxd/storage_pools.go#L714

Should I then also change it there?

Reply:
Ah ok do it as you propose then, but i think we should include Description in both.